pages/1.13/administering-clusters/package-registry/index.md

```diff
@@ -5,15 +5,17 @@ title: Package Registry
 menuWeight: 50
 excerpt: Using the web interface or CLI to manage your package repositories
 enterprise: true
+render: mustache
+model: /data.yml
 ---
 
 <!-- The source repo for this topic is https://github.com/dcos/dcos-docs-site -->
 
-DC/OS comes pre-configured with the [Mesosphere Universe](https://github.com/mesosphere/universe) package repository as the provider of DC/OS packages. However this assumes internet access which is not always possible. For air-gapped environments, DC/OS Enterprise offers the solution of package registry for a flexible and seamless management of your packages.
+DC/OS comes pre-configured with the [Mesosphere {{ model.packageRepo }}](https://github.com/mesosphere/universe) package repository as the provider of DC/OS packages. However this assumes Internet access, which is not always possible. For air-gapped environments, DC/OS Enterprise offers a package registry for a flexible and seamless management of your packages.
```

Comment:
`DC/OS comes pre-configured with the [Mesosphere Universe]` is correct.

I want to clarify couple details:

The button in DC/OS UI is called `Catalog`.
The catalog contains data from `n` package repositories.
DC/OS does come pre-configured with the `Universe`.
User can add more package repositories. `Local Universe`, `Package Registry` are such examples.